media-video/webcamoid/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>reagentoo@gmail.com</email>
+		<name>Dmitry Baranov</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">webcamoid/webcamoid</remote-id>
+	</upstream>
+	<use>
+		<flag name="headers">Install development headers</flag>
+		<flag name="libuvc">Build with libuvc support</flag>
+		<flag name="qtaudio">Enable audio capture/play via <pkg>dev-qt/qtmultimedia</pkg></flag>
+		<flag name="v4l2">Build with Video4Linux v2 support</flag>
```

Comment:
It'd be better to use the global v4l use flag. v4l2 doesn't mean anything different.